### PR TITLE
Spectra v2.10.4

### PR DESCRIPTION
**Pull request by adi3890**
* Security Fix: Hardened the security of the plugin. Props: Wordfence.
* Feature: Container - Added new auto-width and equal-width options to the Container block.
* Improvement: Buttons - Added an option to inherit the secondary styles when using the Astra theme.
* Improvement: Inline Notice - Added accessibility for the close button.
* Improvement: Global - Added viewport width as a typography size option.
* Fix: Container - Resolved an issue wherein the fallback image would appear repeatedly when the Container background was set to video.
* Fix: Form - Resolved an issue wherein form submissions would not work as intended when a ReCaptcha version is used and ReCaptcha is disabled.
* Fix: Image Gallery - Resolved an issue wherein the images in the paginated masonry layout would not be clickable.
* Fix: Popup Builder - Resolved an issue wherein only the latest 10 popups would be visible on the frontend when enabled.
* Fix: Taxonomy List - Resolved an issue wherein the hierarchy would not be displayed as intended in the list layout.

### CODE REVIEW

**Review by rahulvarma722 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 **Requires at least:** 5.6  
 **Requires PHP:** 7.4  
 **Tested up to:** 6.4.1
```
Comment (bsf-zanev) [inline README.md:7]:
```suggestion
**Tested up to:** 6.4.2  
```

File: README.md [hunk 2]
```diff
@@ -233,6 +244,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Fix: Tabs - Resolved an issue in which the tabs wrapper margin would appear differently in the editor and the front-end.
 * Fix: Global - Resolved an issue wherein the WooCommerce and custom template-styles would not load as intended in FSE themes.
 * Fix: Global - Resolved an issue wherein the template styles for custom post types would not load as intended in FSE themes.
+* Feature: Container - Added new auto-width and equal-width options to the Container block.
```
Comment (bsf-zanev) [inline README.md:247]: This should be in 2.10.4.

**Review by yash-tingre-10 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -215,6 +215,17 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
...
+### 2.10.4 - TUESDAY, 12th DECEMBER 2023 ###
+* Security Fix: Hardened the security of the plugin. Props: Wordfence
+* Improvement: Added vw as a Typography size option
+* Improvement: Buttons - Added an option to inherit the secondary styles when using the Astra theme.
+* Improvement: Inline Notice - Added accessibility for the close button.
+* Fix: Container - Resolved an issue wherein the fallback image would appear repeatedly when the Container background was set to video.
+* Fix: Form - Resolved an issue wherein form submission not working, when ReCaptcha version is selected and enable ReCaptcha is disabled.
+* Fix: Image Gallery - Resolved an issue wherein the images in the paginated masonry layout would not be clickable.
+* Fix: Popup Builder - Resolved an issue wherein only the latest 10 popups would be visible on the frontend when enabled.
+* Fix: Taxonomy List - Resolved an issue wherein the hierarchy would not be displayed as intended in the list layout.
```
Comment (bsf-zanev) [inline README.md:227]: Here is a rephrased version of these 9 entries.

```suggestion
* Security Fix: Hardened the security of the plugin. Props: Wordfence.
* Improvement: Buttons - Added an option to inherit the secondary styles when using the Astra theme.
* Improvement: Inline Notice - Added accessibility for the close button.
* Improvement: Global - Added viewport width as a typography size option.
* Fix: Container - Resolved an issue wherein the fallback image would appear repeatedly when the Container background was set to video.
* Fix: Form - Resolved an issue wherein form submissions would not work as intended when a ReCaptcha version is used and ReCaptcha is disabled.
* Fix: Image Gallery - Resolved an issue wherein the images in the paginated masonry layout would not be clickable.
* Fix: Popup Builder - Resolved an issue wherein only the latest 10 popups would be visible on the frontend when enabled.
* Fix: Taxonomy List - Resolved an issue wherein the hierarchy would not be displayed as intended in the list layout.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, but this PR (#3657) should be merged here before releasing.

File: README.md (hunk 3 above)
Comment (bsf-zanev) [reply README.md:227]: This has been addressed in PR #3657, and will be merged here.

File: README.md (hunk 2 above)
Comment (bsf-zanev) [reply README.md:247]: This has been addressed in PR #3657, and will be merged here.

File: README.md (hunk 1 above)
Comment (bsf-zanev) [reply README.md:7]: This has been addressed in PR #3657, and will be merged here.

**Review by rajmohanreddy774 — APPROVED** [review]

**Pull request merged**